plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -125,6 +125,17 @@ func PredicateMetadata(p *v1.Pod, nodeInfo map[string]*schedulercache.NodeInfo)
 	return pm.GetMetadata(p, nodeInfo)
 }
 
+func TestErrorMessageForMissingNode(t *testing.T) {
+	pod := &v1.Pod{}
+	// no node information
+	nodeInfo := &schedulercache.NodeInfo{}
+	detailedNodeMessage := "node not found: &NodeInfo{"
+	_, _, errMsg := PodFitsResources(pod, PredicateMetadata(pod, nil), nodeInfo)
+	if !strings.Contains(fmt.Sprintf("%v", errMsg), detailedNodeMessage) {
```

Comment:
it could be but we don't want to test the internal string rep of NodeInfo here...?